trp_project_scrum/models/project_task_work.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+from openerp import models, fields, api, _
+
+
+class ProjectTaskWork(models.Model):
+    _inherit = "project.task.work"
+
+    @api.model
+    def get_user_related_details(self, user_id):
+        emp_obj = self.env['hr.employee']
+        user = self.env['res.users'].browse(
+            self.env.context.get('user_id') or self._uid
+        )
+        emp_id = emp_obj.search(
+            [('user_id', '=', user.id)],
+            limit=1
+        )
+        if not emp_id and user.is_collaborator:
+            # defaults for therp
```

Comment:
call the functions that produce this defaults: https://github.com/OCA/OCB/blob/8.0/addons/hr_timesheet/hr_timesheet.py#L38